cmd/ci-operator/main.go

```
@@ -601,6 +608,17 @@ func (o *options) initializeNamespace() error {
 		return fmt.Errorf("could not get core client for cluster config: %v", err)
 	}
 
+	routeClient, err := routeclientset.NewForConfig(o.clusterConfig)
+	if err != nil {
+		return fmt.Errorf("could not get route client for cluster config: %v", err)
+	}
+
+	if o.exposeTemplatePod {
+		if err := createTemplateServiceAndRoutes(client, routeClient, o.templates, o.namespace); err != nil {
+			log.Printf("warning: %v", err)
```

does it ignore the error?